sim/core/metrics_aggregator.go

```diff
@@ -308,7 +319,16 @@ func (unitMetrics *UnitMetrics) addSpellMetrics(spell *Spell, actionID ActionID,
 	}
 
 	if !ok {
-		actionMetrics = &ActionMetrics{IsMelee: spell.Flags.Matches(SpellFlagMeleeMetrics)}
+		spellType := proto.SpellType_SpellTypeCast
+		if spell.dots != nil {
```

Comment:
How are the hit count metrics handled for spells with both an initial and periodic damage component, such as Rake / Thrash / Lacerate for Druids? Currently the spell code for "pure DoT" spells like Rip is set up to manually reduce the `Hits` counter by 1 within `ApplyEffects` so that the correct tick count is shown in the UI, and we don't do this for Rake / Thrash / Lacerate by contrast.

Reply:
I haven't really made changes to how the hits work (as I don't posses the BE knowledge for it), but this is how it looks right now:

**Lacerate**
![image](https://github.com/user-attachments/assets/36cfcd40-f285-4c89-8784-04b63353942b)

**Rake / Thrash**
![image](https://github.com/user-attachments/assets/c4c44674-428b-4ffb-a0fa-17f1b3f89191)

Reply:
Also see the comment below, I think we need to track these metrics separately and when having a combined spell that has both we can show it properly as well.

Reply:
This is fixed